schemas/context/experienceevent.example.1.json

```diff
@@ -114,5 +114,23 @@
   },
   "xdm:marketing": {
     "xdm:trackingCode": "marketingcampaign111"
-  }
+  },
+  "xdm:segmentMemberships": [
+    {
+      "xdm:segmentID": {
+        "@id": "https://data.adobe.io/entities/identity/92312748749128",
+        "xdm:namespace": {
+          "xdm:code": "AAM"
+        }
+      },
+      "xdm:lastQualificationTime": "2017-09-26T15:52:25+00:00",
+      "xdm:version": "1.0",
+      "xdm:validUntil": "2017-12-26T15:52:25+00:00",
+      "xdm:status": "realized",
```

Comment:
what's a realized status?

Reply:
@amuraru, it comes from here: https://github.com/harleensahni/xdm/pull/3/files#diff-cd4afc92ff6cef7f5482454896437093R105 

The realized value means that the visitor entered the segment (new)